### PR TITLE
8 patch article by

### PR DESCRIPTION
**Pull request by sirbacharach**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -216,4 +216,63 @@ describe("GET /api/articles/:article_id", () => {
...
+      .expect(404)
+      .then((response) => {
+        const error = response.body.msg;
+        expect(error).toBe("not found");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("202: adds the amount of votes in the given votes object to the total votes of given article_id", () => {
```
Comment (Hard-Luck) [inline __tests__/app.test.js]: `200`, 202 suggests the response is okay but the process incomplete

**Review by Hard-Luck — COMMENTED** [review]
Decent work, just the status code. You also might choose to test if votes can be decremented

File: __tests__/app.test.js (hunk 1 above)
Comment (sirbacharach) [reply __tests__/app.test.js]: Thanks for the feedback, added the test for negative vote value error and also changed response code to 200 for the success test on line 231 of app test.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -216,6 +216,76 @@ describe("GET /api/articles/:article_id", () => {
...
+  test("400: responds with error if votes to be added is a negative value, dissallowing vote reduction", () => {
+    const newVotes = { inc_votes: -2 };
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(400)
+      .send(newVotes)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
```
Comment (Harrison365) [inline __tests__/app.test.js:288]: Nice tests 👍

**Comment by Harrison365** [comment]
Nice work, Rob. 
Merge when ready ✅

**Pull request merged**